TombEngine/Game/Lara/lara_climb.cpp

```diff
@@ -155,7 +155,10 @@ void lara_col_climb_up(ItemInfo* item, CollisionInfo* coll)
 		 
 		if (IsHeld(In::Forward) && resultRight && resultLeft)
 		{
-			if (resultRight < 0 || resultLeft < 0)
+			// Hack: to avoid clipping into angled wall, make sure both probes are not detecting the wall
```

Comment:
Maybe I am wrong, but isn't `||` operation more restrictive than `&&` in this case? If you change to `&&`, it means that both conditions must succeed, not just one.